### DISCUSSION TITLE
vits synthesis

### DISCUSSION THREAD

**Discussion by str20tbl** [coqui-ai/TTS#1065] [General Q&A] [answered]
HI,

I have been training a VITS model and I'm struggling to reproduce the same quality of voice I get from the tensorboard during training.

Given that VITS is an end-to-end implementation I didn't expect to see the `tts` app complain that no vocoder was specified and use the Griffin-Lim 

> Using model: vits
> Using Griffin-Lim as no vocoder model defined

Am I missing a trick on how to synthesise audio using VITS?

**Comment by erogol** [comment, marked as answer]
ignore the complain

**Reply by str20tbl** [reply]
I made the assumption that best_model.pth.tar was the model to use however using a checkpoint model I can successfully replicate the quality of audio.

Thank you for your prompt reply :)
